main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -49,6 +49,7 @@ public static void main(String[] argv) {
         System.out.println("Validators:");
         System.out.println(validatorLoader.listValidators());
 
+        final long startMillis = System.currentTimeMillis();
```

Comment:
Measuring elapsed time on Java should always use `System.nanoTime()`:
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/System.html#nanoTime()

For example see https://www.baeldung.com/java-measure-elapsed-time.

Reply:
That's a great point! Done.